roles/ansible-test/tasks/init_test_options.yaml

```diff
@@ -49,7 +49,7 @@
     ansible_test_options: "{{ ansible_test_options }} --python {{ ansible_test_python }}"
   when:
     - ansible_test_python
-    - not ansible_test_docker
+    - ansible_test_docker_python or (not ansible_test_docker)
```

Comment:
If this is the case, you can just remove this check now, and we'll pass the --python option to the container.  However, when I first set it up this didn't work.

see: https://github.com/ansible/ansible-zuul-jobs/pull/776 for the original support

Reply:
Ok, that works for me, I'd assumed there was a reason why a different collection didn't want to run with the limits